### ISSUE TITLE
ip location

### ISSUE THREAD

**Issue by alialpha123** [thelinuxchoice/locator#1]
1st problem is that we dont get any link when we select Ngrock and 2nd problem is that when we get site link from 1st option so when we press enable it for location on browsers so in terminal we get user denied geolocation

**Comment by vepicurean** [comment]
Agreed on the ngrok option, no link is shown to send to the target. 

And on the serveo port forward option, most people will not automatically agree to accept the geolocation notification in the browser.  Serveo is an excellent tool. 

Suggestion.  Similar to "Trape".  It would be good to have the ability to add the url for a specific web page that you want the target to go to.  These pages are too generic. Most people will fit it a strange page to receive a link to view.

**Comment by billyst12** [comment]
bro, how did you solve "can't get any link issue"? thank you.

**Comment by doophack** [comment]
it was a simple script problem , i really forgot what is the problem . 
but i have a copy of {locator} #Fixed
and Here's ! 
[locator2.zip](https://github.com/thelinuxchoice/locator/files/3105291/locator2.zip)
Extract Zip Have Fun :)
 And Thanks To TheLinuxChoice <3

**Comment by WhisperX1110** [comment]
> it was a simple script problem , i really forgot what is the problem .
> but i have a copy of {locator} #Fixed
> and Here's !
> [locator2.zip](https://github.com/thelinuxchoice/locator/files/3105291/locator2.zip)
> Extract Zip Have Fun :)
> And Thanks To TheLinuxChoice <3

it still doesn't work for me

**Comment by WhisperX1110** [comment]
help me please

**Comment by jusondac** [comment]
how to fix it.

**Comment by EulerIExponent** [comment]
> how to fix it.

This tool only work with cellular data .Hence turn off your wifi start cellular data and hotspot
